docs/architecture.md

```diff
@@ -105,8 +105,8 @@ receive commands from Kubernetes, via a CRI client to launch and run Pods.
 
 ### Cloud API Adaptor
 The [cloud-api-adaptor](../cmd/cloud-api-adaptor) implements the remote hypervisor. It runs as a daemonset on
-each node and is responsible for receiving commands from the `containerd-shim-kata-v2 process` and implementing them
-for peer pods. More example, when it recieved commands that were related to Pod lifecycle that are traditionally
+each node and is responsible for receiving commands from the `containerd-shim-kata-v2` process and implementing them
+for peer pods. Fo example, when it receives commands that were related to Pod lifecycle that are traditionally
```

Comment:
s/Fo/For

Last comment :)

Reply:
No Friday excuse for this one - just a virtual Monday!